nautobot/extras/templates/extras/approval_workflow/approve.html

```diff
@@ -1,11 +1,15 @@
 {% extends 'utilities/confirmation_form.html' %}
 {% load form_helpers %}
+{% load helpers %}
 
-{% block title %}Approve {{ obj_type }}?{% endblock %}
+
+{% block title %}Approve Workflow Stage?{% endblock %}
 
 {% block message %}
     {% block message_extra %}{% endblock %}
-    <p>Are you sure you want to approve <strong>{{ obj }}</strong> for <strong> {{ object_under_review }}</strong>?</p>
+    <p>Are you sure you want to approve</p>
+    <ul><li>Stage <strong>{{ obj.approval_workflow_stage_definition.name }}</strong></li><li>from Workflow <strong>{{ obj.approval_workflow_stage_definition.approval_workflow_definition.name }}</strong></li>
+        <li>for {{ object_under_review|meta:"verbose_name"|bettertitle }} <strong>{{ object_under_review|hyperlinked_object}}</strong>?</li></ul>
```

Comment:
djlint/djhtml might do this for us, but suggest:

```suggestion
    <ul>
        <li>Stage <strong>{{ obj.approval_workflow_stage_definition.name }}</strong></li>
        <li>from Workflow <strong>{{ obj.approval_workflow_stage_definition.approval_workflow_definition.name }}</strong></li>
        <li>for {{ object_under_review|meta:"verbose_name"|bettertitle }} <strong>{{ object_under_review|hyperlinked_object}}</strong>?</li>
    </ul>
```